go/vt/vttablet/tabletmanager/vreplication/vplayer_flaky_test.go

```diff
@@ -1270,7 +1270,10 @@ func TestPlayerRowMove(t *testing.T) {
 
 func TestPlayerTypes(t *testing.T) {
 	log.Errorf("TestPlayerTypes: flavor is %s", env.Flavor)
-
+	enableJSONColumnTesting := false
+	if strings.EqualFold(env.Flavor, "mysql57") {
```

Comment:
We should be testing mysql80 as well.

Reply:
If this is something that doesn't work only on percona56, we should exclude just that one flavor.

Reply:
Added mysql80 in addition to mysql57. The Percona flavor is mysql56 and other platforms start with mariadb.